build/scripts/version-update/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
```

Comment:
From linter: ` SA1019: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)`

Reply:
replaced io/ioutil with os👍🏻